Cargo.toml

```diff
@@ -12,9 +12,12 @@ crate-type = [
 ]
 
 [dependencies]
+getrandom = "0.2.15"
```

Comment:
Isn't this a dev-only dependency? We already have a `rand` dependency in `dev-dependencies`.

Note, that `dev-dependencies` are present for unit tests.

Reply:
Dropped those unneeded dependencies in favor of `rand` by using:

```rust
StdRng::from_entropy().fill_bytes(&mut random_data);
```